pages/linux/gamescope.md

```diff
@@ -0,0 +1,16 @@
+# gamescope
+
+> Micro-compositor used as a game layer.
+> More information: <https://github.com/ValveSoftware/gamescope>.
+
+- Upscale a 720p game to 1440p with integer scaling:
+
+`gamescope -h 720 -H 1440 -S integer -- %command%`
```

Comment:
This doesn't seem to be documented anywhere so you have to dig into the code https://github.com/ValveSoftware/gamescope/blob/master/src/main.cpp
```suggestion
`gamescope {{[-h|--nested-height]}} 720 {{[-H|--output-height]}} 1440 {{[-S|--scaler]}} integer -- %command%`
```